clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -202,6 +202,30 @@ def IntAttr : CIR_Attr<"Int", "int", [TypedAttrInterface]> {
   let hasCustomAssemblyFormat = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// FloatAttr
+//===----------------------------------------------------------------------===//
+
+def FloatAttr : CIR_Attr<"Float", "float", [TypedAttrInterface]> {
```

Comment:
Let's call this `FPAttr`, the reason is that `FloatAttr` might resemble specific fp representation f32. In CIR it should read like `#cir.fp<...>`

Reply:
Updated.